src/Mobs/MagmaCube.h

```diff
@@ -19,10 +19,14 @@ class cMagmaCube :
 
 	virtual void GetDrops(cItems & a_Drops, cEntity * a_Killer = nullptr) override;
 	int GetSize(void) const { return m_Size; }
+
+	/** Returns the text describing the slime's size, as used by the client's resource subsystem for sounds.
+	Returns either "big" or "small". */
+	const AString GetSizeName(int a_Size) const;
```

Comment:
Would you like me to create a commit making GetSizeName of Slime and MagmaCube static?